src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -2462,6 +2466,20 @@ protected function get_schema_links() {
 			);
 		}
 
+		$links[] = array(
+			'rel'          => 'https://api.w.org/action-delete',
+			'title'        => __( 'The current user can delete this post.' ),
+			'href'         => $href,
+			'targetSchema' => array(
+				'type'       => 'object',
+				'properties' => array(
+					'delete' => array(
+						'type' => 'boolean',
+					),
+				),
```

Comment:
This probably needs to be something else?

Reply:
The idea of the `targetSchema` is so that a developer can programmatically determine that changing the resource such that it would successfully validate against the `targetSchema` would require a specific set of capabilities beyond `PUT` being available in the `Allow` header. The developer can then verify that the resource includes this link in the response.

Since a post is deleted by making a `DELETE` request instead of a `PUT`, the `targetSchema` can't actually be used and the link relation solely communicates the semantics.

So we could drop the `targetSchema` entirely.